src/Admin/AbstractAdmin.php

```diff
@@ -3265,10 +3265,7 @@ public function canAccessObject($action, $object)
         return $this->hasAccess($action, $object);
     }
 
-    /**
-     * @return MutableTemplateRegistryInterface
-     */
-    final public function getTemplateRegistry()
+    final public function getTemplateRegistry(): ?MutableTemplateRegistryInterface
```

Comment:
Why it can return null?

Reply:
I can in 3.x.

It won't in master. A next major would be needed indeed.